src/GraphQL.Harness/appsettings.Development.json

```diff
@@ -1,4 +1,4 @@
-﻿{
+{
```

Comment:
ooops, BOM even here
https://stackoverflow.com/questions/4990095/json-specification-and-usage-of-bom-charset-encoding

Reply:
Don’t care. Works fine. Absolutely no effect on the functionality of this sample program, which isn’t even included in this library.